test/functional/plugin/lsp_spec.lua

```diff
@@ -6121,6 +6121,22 @@ describe('LSP', function()
     end)
   end)
 
+  describe('vim.lsp.start', function()
+    it('can launch when an existing client has no workspace_folders', function()
```

Comment:
The whole file is testing `vim.lsp.start` so this describe() is a bit unnecessary. Can just be a it() test.
```suggestion
  it('vim.lsp.start when existing client has no workspace_folders', function()
```